packages/material-ui/package.json

```diff
@@ -50,10 +50,10 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.4",
-    "@material-ui/styles": "^4.11.4",
-    "@material-ui/system": "^4.12.1",
+    "@material-ui/styles": "4.11.5",
```

Comment:
Why not:
```suggestion
    "@material-ui/styles": "^4.11.5",
```

Reply:
Ah forgot `yarn release:version` removes the ranges